File: Java/src/Act2/Habitacion.java
package Act2;

public class Habitacion {
    private int numero;
    private boolean disponible;

    public Habitacion(int numero, boolean disponible) {
        this.numero = numero;
        this.disponible = true;
    }

    public int getNumero() {
        return numero;
    }

    public boolean isDisponible() {
        return disponible;
    }

    public void ocupar() {
        disponible = false;
    }

    public void liberar() {
        disponible = true;
    }

    @Override
    public String toString() {
        return "Habitacion{" +
                "numero=" + numero + '\'' +
                ", disponible=" + disponible +
                '}';
    }
}

File: KotlinCode/src/Act2/Habitacion.kt
package Act2

class Habitacion(var numero: Int, var disponible: Boolean) {

    fun isDisponible(): Boolean {
        return disponible
    }

    fun ocupar() {
        disponible = false
    }

    fun liberar() {
        disponible = true
    }

    override fun toString(): String {
        return "Habitacion{" +
                "numero=" + numero + '\'' +
                ", disponible=" + disponible +
                '}'
    }
}

